webui/react/src/components/NotesCard.tsx

```diff
@@ -52,6 +52,31 @@ const NotesCard: React.FC<Props> = ({
     setEditedNotes(existingNotes.current);
   }, [noteChangeSignal]);
 
+  useEffect(() => {
```

Comment:
alternative of Prompt cuz new `react-router-dom` doesnt have `prompt` yet

Reply:
✅  tested editing note and getting ok/cancel out